### PR TITLE
Add grey line below nav bar, change logo img max height 44px

### LINKED ISSUE #4034: Add grey line in main header
## Description
Yuko's header design was showing a grey line at the bottom of the main header.

Very useful when you scroll on a white background.

## Expected Behavior

image

## Actual Behavior

image

## Steps to Reproduce

Just scroll on any page on a instance on v2.2 and onwards

## Your Environment
* Version used: v2.2
* Browser name and version: Any
* Operating System and version (desktop or mobile): Any

### PR DESCRIPTION
**Pull request by idanceinbetween**
#### What? Why?

Closes #4034 

#### What should we test?
Visually check grey line (use `$ofn-grey`) is present as expected. Use inspector to check logo max height is 44px. 

#### Release notes
Grey line has been added below navbar. Logo's img max height has been changed from 51px to 44px based on https://github.com/openfoodfoundation/openfoodnetwork/issues/4034#issuecomment-510927335

Changed

### CODE REVIEW

File: app/assets/stylesheets/darkswarm/menu.css.scss [hunk 1]
```diff
@@ -1,8 +1,8 @@
-@import "compass";
-@import "branding";
-@import "mixins";
-@import "typography";
-@import "variables";
```
Comment (luisramos0) [inline app/assets/stylesheets/darkswarm/menu.css.scss]: we are using double quotes in all css files right now, I'd keep the local dialect.

Comment (idanceinbetween) [reply app/assets/stylesheets/darkswarm/menu.css.scss]: Yes, glad to be back as I've finally graduated and found a job 😄 Sorry about the quotation marks - it was done by my Prettier, I'll remember to disable it next time!

Comment (luisramos0) [reply app/assets/stylesheets/darkswarm/menu.css.scss]: new job, congrats!

**Comment by RachL** [comment]
Awesome to have you back @idanceinbetween ! :tada:

And thank you for tackling this. Having a header that looks like falling was terribly awful for me :D

The PR is good to merge. However may I ask how you've chosen the grey color?

On Yuko's design, the grey looks like the same grey as the one in the line separating the language and cart button:

image

But after staging your PR, it seems that the grey below the header is darker (testing on Firefox / Edge / Chrome):

image

Is it the same grey (and my computer screen sucks) or are they different grey?

If they are different and if you don't have time to fix this, let's merge. Just let me know :)

**Comment by idanceinbetween** [comment]
> Awesome to have you back @idanceinbetween ! 🎉
> 
> And thank you for tackling this. Having a header that looks like falling was terribly awful for me :D
> 
> The PR is good to merge. However may I ask how you've chosen the grey color?
> 
> On Yuko's design, the grey looks like the same grey as the one in the line separating the language and cart button:
> 
> image
> 
> But after staging your PR, it seems that the grey below the header is darker (testing on Firefox / Edge / Chrome):
> 
> image
> 
> Is it the same grey (and my computer screen sucks) or are they different grey?
> 
> If they are different and if you don't have time to fix this, let's merge. Just let me know :)

oops sorry @RachL just saw your message :( I've chosen $ofn-grey because that seemed to be the only colour that's mentioned in the stylesheet and would pass the test. Yeh it's a bit darker I can see on your screenshot. But since it's been merged now, lemme know if it should be changed to a lighter one!

**Comment by RachL** [comment]
@idanceinbetween if you are interested. We recently made the navbar sticky on mobile as well and noticed the grey line was missing there as well. So I've open https://github.com/openfoodfoundation/openfoodnetwork/issues/4082 to tackle both changing the grey color and doing the same thing you did here but on mobile display as well.

**Pull request merged**